ui/src/components/Header/Header.tsx

```diff
@@ -10,20 +10,23 @@ export const Header = (): ReactElement => {
   return (
     <AppBar position="relative" elevation={0}>
       <Toolbar>
-        <Box display='flex' flexGrow={1}>
-          <Typography variant='h3' sx={{ my: 2, mr: 3 }}>
+        <Box display="flex" flexGrow={1} alignItems="center" flexWrap="wrap">
+          <Typography variant="h3" sx={{ my: 2, mr: 3 }}>
             LocalStack
           </Typography>
-          <Button
-            color="inherit"
```

Comment:
Removing `color="inherit"` makes the button disappear for both light and dark mode (I guess it's the same colour as the background)
![image](https://user-images.githubusercontent.com/36865043/208686395-19041c52-418a-4139-82c7-23c97fb957f5.png)

Reply:
We try to use the `variant` prop, which should have everything correctly set up for each variation. In [095284e](https://github.com/localstack/localstack-docker-extension/pull/17/commits/095284e9fc56e86487d9f35ff2467a38b53b3a6e) I changed the variant to an outlined one, which will stand out a little bit more.
<img width="319" alt="image" src="https://user-images.githubusercontent.com/8645841/208727194-0a0f00a4-b9f9-4baa-811a-4c8d8c337249.png">

Based on your screenshot I cannot tell 100%, do you have the latest Docker Desktop version? We made a UI update in 4.15 to match our Design System, maybe the button disappearing was an old issue (I'm not sure), let me know if it is still happening on the latest DD version.

Reply:
Oh, I see the problem...I was still using an old version because an update made problems with the hot reload for me  (I think v4.12)...Updated to 4.15 and everything works fine :+1: